integration/tests/admin/command_runner_test.go

```diff
@@ -473,17 +462,18 @@ func (suite *CommandRunnerSuite) TestTLS() {
 
 	suite.SetupCommandRunner(admin.WithTLS(serverConfig))
 
-	client := &http.Client{
+	c := &http.Client{
```

Comment:
s/c/client/g

(avoid using single, or even two, character(s) variables in general.  it's hard to search/replace single character variables)